source/backups/creating-backup.rst

```diff
@@ -0,0 +1,92 @@
+################################
+Creating a backup
+################################
+
+The purpose of this section is to provide examples on how you can use different
+tools to create a volume backup on the Catalyst Cloud.
+
+Before you continue with the examples below, there are a few assumptions that
+are made which you will need to consider before jumping in further:
+
+1)
+ You are familiar with the Linux command line and Openstack CLI tools.
+2)
+ You have installed the OpenStack command line tools and sourced an openrc
+ file, as explained in :ref:`this section of the documentation<command-line-interface>`.
+
+
+***********************
+Using the Openstack CLI
+***********************
+
+To create a backup using the openstack command line tools, we first need to
+find the original volume we are trying to back up. To show a list of the
+currently available volumes, you can use the following code snippet:
+
+.. code-block:: bash
+
+    $ openstack volume list
+    +--------------------------------------+---------------------+-----------+------+-----------------+
+    | ID                                   | Name                | Status    | Size | Attached to     |
+    +--------------------------------------+---------------------+-----------+------+-----------------+
+    | 81599985-XXXX-XXXX-XXXX-XXXXXXXXXXee | backup-vol-original | available | 5    |                 |
+    +--------------------------------------+---------------------+-----------+------+-----------------+
+    # we can then export our volume ID for later use:
+    $ export volumeID='81599985-XXXX-XXXX-XXXX-XXXXXXXXXXee'
+
+Once we have the volume that we want to back up we can construct our backup
+command. The syntax for creating a backup using the openstack CLI is:
+
+.. code-block:: bash
+
+    $ openstack volume backup create [--incremental] [--force] <VOLUME>
```

Comment:
It would be nice to have a note for the two parameters  --incremental  and --force.

Especially for the --force, which is used when the volume is in **in-use** status.